Documentation.docc/Documentation.md

```diff
@@ -0,0 +1,116 @@
+🖥 | macOS support
+🎯 | Subscription status tracking - know whether a user is subscribed whether they're on iOS, Android or web
+📊 | Analytics - automatic calculation of metrics like conversion, mrr, and churn
+📝 | [Online documentation](https://docs.revenuecat.com/docs) up to date
+🔀 | [Integrations](https://www.revenuecat.com/integrations) - over a dozen integrations to easily send purchase data where you need it
+💯 | Well maintained - [frequent releases](https://github.com/RevenueCat/purchases-ios/releases)
+📮 | Great support - [Help Center](https://community.revenuecat.com)
+
+## Getting Started
+For more detailed information, you can view our complete documentation at [docs.revenuecat.com](https://docs.revenuecat.com/docs).
+
+Or browse our iOS sample apps:
+- [MagicWeather](Examples/MagicWeather)
+- [MagicWeather SwiftUI](Examples/MagicWeatherSwiftUI)
+
+## Topics
+
+### Purchases
+- ``Purchases``
+
+### Configuring the SDK
+
+- ``Purchases/configure(withAPIKey:)``
+- ``Purchases/configure(withAPIKey:appUserID:)``
+- ``Purchases/configure(withAPIKey:appUserID:observerMode:)``
+- ``Purchases/configure(withAPIKey:appUserID:observerMode:userDefaults:)``
+
+### Displaying Products
+- ``Offerings``
+- ``Offering``
+- ``Package``
+- ``StoreProduct``
+- ``SubscriptionPeriod``
+
+- ``Purchases/getOfferings(completion:)``
+- ``Purchases/getProducts(_:completion:)``
+
+### Making Purchases
+- ``StoreTransaction``
+- ``Purchases/purchase(package:)``
+- ``Purchases/purchase(package:completion:)``
+- ``Purchases/purchase(product:)``
+- ``Purchases/purchase(product:completion:)``
+
+### Making Purchases with Promotional Offers
+- ``IntroEligibility``
+- ``StoreProductDiscount``
+- ``Purchases/checkTrialOrIntroductoryPriceEligibility(_:)``
+- ``Purchases/purchase(package:discount:completion:)``
+- ``Purchases/purchase(product:discount:completion:)``
+
+### Subscription Status
+- ``CustomerInfo``
+- ``EntitlementInfo``
+- ``EntitlementInfos``
+- ``PurchasesDelegate``
+- ``Purchases/getCustomerInfo(completion:)``
+
+### Identifying Users
+- ``Purchases/logIn(_:completion:)``
+- ``Purchases/logOut(completion:)``
+
+### Managing Subscriptions
+- ``Purchases/syncPurchases()``
+- ``Purchases/restorePurchases()``
+- ``Purchases/beginRefundRequestForActiveEntitlement()``
+- ``Purchases/beginRefundRequest(forEntitlement:)``
+- ``Purchases/beginRefundRequest(forProduct:)``
+- ``Purchases/showManageSubscriptions()``
+
+### Subscriber Attributes
+
+- ``Purchases/setAttributes(_:)``
+- ``Purchases/setAd(_:)``
+- ``Purchases/setEmail(_:)``
+- ``Purchases/setDisplayName(_:)``
+- ``Purchases/setKeyword(_:)``
+- ``Purchases/setCampaign(_:)``
+- ``Purchases/setCreative(_:)``
+- ``Purchases/setAdGroup(_:)``
+- ``Purchases/setPushToken(_:)``
+- ``Purchases/setMediaSource(_:)``
+- ``Purchases/setPhoneNumber(_:)``
+- ``Purchases/setAttributes(_:)``
+
+### Integrations
```

Comment:
I'm willing to admit that I got too into this if you're willing to admit this looks cool
<img width="438" alt="image" src="https://user-images.githubusercontent.com/3922667/150363339-7b5b8534-525d-4d56-b774-c2aa43b84cce.png">
<img width="1031" alt="Screen Shot 2022-01-20 at 11 57 58 AM" src="https://user-images.githubusercontent.com/3922667/150363427-7e001150-8aaa-4237-ba75-1608b3984158.png">